checker/tests/resourceleak/Generics.java

```diff
@@ -0,0 +1,15 @@
+import java.util.*;
```

Comment:
How is this test case related to this PR?

Reply:
At one point during development I thought I had a bug related to generics. It turned out to be nothing. This test had some sketchy code to help me figure out what was going on, and now serves no purpose. I'll remove it.